app/src/main/java/app/revanced/manager/ui/screen/settings/AdvancedSettingsScreen.kt

```diff
@@ -138,16 +138,38 @@ fun AdvancedSettingsScreen(
             )
 
             GroupHeader(stringResource(R.string.debugging))
+            val exportDebugLogsLauncher =
+                rememberLauncherForActivityResult(ActivityResultContracts.CreateDocument("text/plain")) {
+                    it?.let(vm::exportDebugLogs)
+                }
             SettingsListItem(
-                headlineContent = stringResource(R.string.about_device),
-                supportingContent = """
-                    **Version**: ${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})
-                    **Build type**: ${BuildConfig.BUILD_TYPE}
-                    **Model**: ${Build.MODEL}
-                    **Android version**: ${Build.VERSION.RELEASE} (${Build.VERSION.SDK_INT})
-                    **Supported Archs**: ${Build.SUPPORTED_ABIS.joinToString(", ")}
-                    **Memory limit**: $memoryLimit
+                headlineContent = stringResource(R.string.debug_logs_export),
+                modifier = Modifier.clickable { exportDebugLogsLauncher.launch(vm.debugLogFileName) }
+            )
+            val clipboard = remember { context.getSystemService(CLIPBOARD_SERVICE) as ClipboardManager }
```

Comment:
Use the extension with this signature: `context.getSystemService<ClipboardManager>()!!` for the sake of consistency.

Reply:
Done